lib/ansible/executor/shared_plugin_loader.py

```diff
@@ -18,3 +18,22 @@
 # Make coding more python3-ish
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
+
+from abc import ABCMeta
+
+from ansible.module_utils.six import with_metaclass
+from ansible.plugins.loader import action_loader, connection_loader, filter_loader, lookup_loader, module_loader, test_loader
+
+
+class SharedPluginLoaderObj(with_metaclass(ABCMeta, object)):
```

Comment:
still not sure why we need this, just `from ansible.plugins import loader as shared_loader` seems to do same thing?